planner/go_planner.go

```diff
@@ -39,6 +51,31 @@ func (g *GoPlanner) GetPlan(srcDir string) *Plan {
 	}
 }
 
+func getGoPackage(srcDir string) string {
+	goModPath := filepath.Join(srcDir, "go.mod")
+	goVersion := getVersion(goModPath)
+	v, ok := versionMap[goVersion]
+	if ok {
+		return v
+	} else {
+		// Should we be throwing an error instead, if we don't have a nix package
+		// for the specified version of go?
+		return defaultPkg
+	}
+}
+
+func getVersion(gomodPath string) string {
```

Comment:
I think `getVersion` is kind of vague because this lives in the same package as other planners. Maybe `parseGoVersion`?

Reply:
Can also be a method on `GoPlanner` 

python will have similar

Reply:
Changing to `parseGoVersion` as requested.